ArchiSteamFarm/Bot.cs

```diff
@@ -2826,6 +2836,127 @@ internal sealed class Bot : IDisposable {
 			return responses.Count > 0 ? string.Join("", responses) : null;
 		}
 
+
+		private async Task<string> ResponseLootByRealAppID(ulong steamID, string targetRealAppIDs) {
+		        if ((steamID == 0) || string.IsNullOrEmpty(targetRealAppIDs)){
+			        ArchiLogger.LogNullError(nameof(steamID) + " || " + nameof(targetRealAppIDs));
+				return null;
+			}
+
+			if (!IsMaster(steamID)) {
+				return null;
+			}
+
+			if (!IsConnectedAndLoggedOn) {
+				return FormatBotResponse(Strings.BotNotConnected);
+			}
+
+			string[] appIDText = targetRealAppIDs.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries);
+
+	                if (appIDText.Length == 0) {
+				return FormatBotResponse(string.Format(Strings.ErrorIsEmpty, nameof(appIDText)));
+			}
+
+			HashSet<uint> realAppIDs = new HashSet<uint>();
+
+	                foreach (string id in appIDText){
+			    if (!uint.TryParse(id, out uint appID) || (appID == 0)) {
+				return FormatBotResponse(string.Format(Strings.ErrorIsInvalid, nameof(appID)));
+			    }
+			    realAppIDs.Add(appID);
+			}
+
+			if (!LootingAllowed) {
+				return FormatBotResponse(Strings.BotLootingTemporarilyDisabled);
+			}
+
+			if (BotConfig.LootableTypes.Count == 0) {
+				return FormatBotResponse(Strings.BotLootingNoLootableTypes);
+			}
+
+			ulong targetSteamMasterID = GetFirstSteamMasterID();
+			if (targetSteamMasterID == 0) {
+				return FormatBotResponse(Strings.BotLootingMasterNotDefined);
+			}
+
+			if (targetSteamMasterID == CachedSteamID) {
+				return FormatBotResponse(Strings.BotSendingTradeToYourself);
+			}
+
+			lock (LootingSemaphore) {
+				if (LootingScheduled) {
```

Comment:
You don't need `LootingScheduled` because `LootingSemaphore` already guarantees maximum of one request, scheduled looting is needed only if the command can be executed by events, this one is not. All `locks` should be removed.